onnxruntime/test/onnx/TestCase.cc

```diff
@@ -16,6 +14,8 @@
 #include <map>
 #include <regex>
 #include "OrtValueList.h"
+#undef OPTIONAL  // to get around onnxruntime\cmake\external\onnx\onnx/common/constants.h(14): error C2513: 'bool': no variable declared before '='
```

Comment:
You may also workaround it by reordering the header files.

Reply:
fixed.